### PR TITLE
feat: template offline install.

### PR DESCRIPTION
**Pull request by lingdie**
Template offline install.
1. init container
2. images used in templates will be build into cluster image

### CODE REVIEW

**Comment by lingdie** [comment]
please add `secrets.GH_PAT`.

File: docker/init.dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM bitnami/git
+WORKDIR /data
+RUN git clone https://github.com/labring-actions/templates.git
```
Comment (zzjin) [inline docker/init.dockerfile:3]: using `--depth=1` to shim image size.

Comment (zzjin) [reply docker/init.dockerfile:3]: can this also support branch name?

Comment (lingdie) [reply docker/init.dockerfile:3]: > can this also support branch name?

run command `git checkout`？

Comment (lingdie) [reply docker/init.dockerfile:3]: > using `--depth=1` to shim image size.

ok

**Review by zzjin — APPROVED** [review]

**Pull request merged**